### DISCUSSION TITLE
Tasks for the weeks 11/27 - 12/4

### DISCUSSION THREAD

**Discussion by kaushikjadhav01** [NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11#44] [General]
Hi All,

Please post your updates for this week's tasks below.

Thanks & Regards,
Kaushik Jadhav

**Comment by kaushikjadhav01** [comment]
1. Fix Android Tests
2. Fix Github Actions
3. Increase Test Coverage
4. Release v2.0
5. Shared Users in Edit Transactions

**Comment by aditi-v79** [comment]
Tasks:
1. Set user limits for day/month/year
2. Sending user notification when limit is exceeded. 
3. Updating the README accordingly.

**Comment by vemuriPradyumna** [comment]
The SMSDBFetchActivity Class and its view do not display the shared email list for transactions, even though they are visible in the main menu. I have created an issue for the same.

**Comment by kaushikjadhav01** [comment]
Issue raised in: https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/41

**Comment by kaushikjadhav01** [comment]
Fixed issue https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/41 in PR https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/pull/50

**Comment by surya-prakash-susarla** [comment]
Will be fixing all the doi/ coverage status report links in the readme files. will try to refactor the code in some places to see if we can improve any metrics for the crossfold factor.

**Comment by vemuriPradyumna** [comment]
Tasks:

- Adding style/code checkers as action in Workflows.
- Make a demo video for the application.
- Generate and host documentation for the code.
- Update or Add the project rubrics.

**Comment by surya-prakash-susarla** [comment]
Added a few more tasks for the documentation part and other enhancements related to the config files for style checkers. Please take a look at the updated tasks. Thanks! cc: @vemuriPradyumna

**Comment by vemuriPradyumna** [comment]
Tasks:

- Added a scope file to include only main XML and Java classes while performing a manual inspection.
- Added a .clang-format config file to maintain consistent styling.

**Comment by Hp-Harshitha-hp** [comment]
Tasks:

- Have implemented Analytics which gives visualizations with the given input data
- Have implemented 2 options for visualizations with 2 different inputs
-        1. Day-wise visualizations in a specific month
-         2. Month-wise visualizations
- update README accordingly
